enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/iverksett/PeriodebeløpDto.kt

```diff
@@ -1,14 +0,0 @@
-package no.nav.familie.kontrakter.ef.iverksett
-
-import java.time.LocalDate
-
-data class PeriodebeløpDto(
-        val beløp: Int,
-        var periodetype: Periodetype,
-        val fraOgMed: LocalDate,
-        val tilOgMed: LocalDate,
-)
-
-enum class Periodetype {
```

Comment:
Vi har kun måned nå, men tror vi kommer få engangsutbetaling for noen av de andre ytelsene

Reply:
Ser inte vitsen att ef-sak ska senda det nå når vi inte har några andra ytelser dock..